src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    men = men.filter(man => Math.ceil(man.died / 100) === century);
+  }
+
+  const averageAge
+  = men.reduce(
+    (prev, man) => prev + man.died - man.born, 0)
+  / men.length;
+
```

Comment:
Great job! I like the way you solve it! However there are few thing to fix, mostly your formatting. I'm not sure, but I guess   closing parenthesis shoud be at the same level with string you started method.